File: notify/v1/segment_membership/create-add-user-to-segment/create-add-user-to-segment.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$sid    = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$segment_membership = $twilio->notify->v1->services("ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                                         ->users("NUaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
                                         ->segmentMemberships
                                         ->create("premium");

print($segment_membership.accountSid);

File: notify/v1/segment_membership/create-add-user-to-segment/create-add-user-to-segment.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

segment_membership = @client.notify
                            .services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                            .users('NUXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                            .segment_memberships
                            .create(segment: 'premium')

puts segment_membership.account_sid
